projects/demo/src/modules/components/select/examples/9/account/my-account.component.html

```diff
@@ -0,0 +1,25 @@
+<ng-container *tuiLet="account | tuiMapper: mapper as model">
```

Comment:
This is excessive, let's make it a simple representation of the idea that you can store content as reusable components, not a complete copy and paste of our full fledged account component 🙂